### ISSUE TITLE
[Magento 2] Executing magento command gives IDE error

### ISSUE THREAD

**Issue by LordZardeck** [enriquepiatti/Magicento#269]
I get the following error when executing magento commands, it appears to be any command:

```
com.jetbrains.php.run.PhpExecutionUtil.getProcessOutput(Lcom/intellij/openapi/project/Project;Lcom/jetbrains/php/config/interpreters/PhpSdkAdditionalData;Lcom/intellij/execution/configurations/GeneralCommandLine;Ljava/lang/String;Ljavax/swing/JComponent;IZ)Lcom/intellij/execution/process/ProcessOutput;
java.lang.NoSuchMethodError: com.jetbrains.php.run.PhpExecutionUtil.getProcessOutput(Lcom/intellij/openapi/project/Project;Lcom/jetbrains/php/config/interpreters/PhpSdkAdditionalData;Lcom/intellij/execution/configurations/GeneralCommandLine;Ljava/lang/String;Ljavax/swing/JComponent;IZ)Lcom/intellij/execution/process/ProcessOutput;
	at com.magicento2.helpers.PHP.executeWithPhpInterpreter(PHP.java:426)
	at com.magicento2.actions.ExecuteMagento2CommandAction.executeAction(ExecuteMagento2CommandAction.java:48)
	at com.magicento2.actions.Magicento2ActionAbstract.actionPerformed(Magicento2ActionAbstract.java:424)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:216)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:233)
	at com.intellij.ide.actions.GotoActionAction.a(GotoActionAction.java:337)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:109)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:118)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:270)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.a(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

I'm using a remote interpreter if that makes a difference

**Comment by enriquepiatti** [comment]
what version of Magicento and PhpStorm are you using?

**Comment by LordZardeck** [comment]
Plugin says it's on 1.6.0 here's my PHPStorm details:

PhpStorm 2017.2.4
Build #PS-172.4155.41, built on September 15, 2017
JRE: 1.8.0_152-release-915-b11 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.12.6

**Comment by enriquepiatti** [comment]
OK, I can see the definition for PhpExecutionUtil.getProcessOutput has changed in 2017.2, I will add a fix to manage that too.
Maybe you can disable the PHP features for now if you need them with PHP interpreter (or change the method) and the error is giving you some problems

**Comment by enriquepiatti** [comment]
fixed in v1.7.0
